agoradatatools/process.py

```diff
@@ -107,26 +109,36 @@ def process_all_files(config_path: str = None, syn=None):
     else:
         config = utils._get_config()
 
-    datasets = config[1]["datasets"]
-
+    datasets = utils._find_config_by_name(config, "datasets")
+    
     # create staging location
-    load.create_temp_location()
-
+    staging_path = utils._find_config_by_name(config, "staging_path")
+    if staging_path is None: 
+        staging_path = "./staging"
+        
+    load.create_temp_location(staging_path)
+    
     if datasets:
         for dataset in datasets:
             new_syn_tuple = process_dataset(
-                dataset_obj=dataset, syn=syn
+                dataset_obj=dataset, staging_path=staging_path, syn=syn
             )
             # in the future we should log new_syn_tuples that are none
 
+    destination = utils._find_config_by_name(config, "destination")
```

Comment:
There are some key redactors that I want to do with the config that will basically enable us to not have to use this function.

TLDR, @JessterB knows which Jira ticket it is, but it is the one to add yaml anchors for all the synapse ids in the config.

Great idea for the meantime!